tests/test_apigw.py

```diff
@@ -214,3 +214,17 @@ def test_rest_stage_update(self):
             self.assertEqual(m["loggingLevel"], "INFO")
             self.assertEqual(m["dataTraceEnabled"], True)
         self.assertTrue(found)
+
+    def test_rest_stage_delete(self):
+        session_factory = self.replay_flight_data("test_rest_stage_delete")
+        p = self.load_policy(
+            {
+                "name": "rest-stage-delete",
+                "resource": "rest-stage",
+                "filters": [{"type": "value", "key": "stageName", "value": "testing-skunk-1"}],
+                "actions": [{"type": "delete"}],
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 0)
```

Comment:
This test doesn't really exercise the delete action. zero resources means zero actions taken.

Reply:
@kapilt Any suggestions to write this test better? When a delete operation is successful against the apigateway api it just returns a 200 ok and no resultset.

Reply:
this policy didn't even delete anything, no actions were executed... in general on delete you can verify the output of a describe/get afterwards to check the resource state/nonexistence.

Reply:
Adding missing deletestage API call, updated check of deleted resource using try except check.